cosmos/loadtest/scripts/loadtest.sh

```
@@ -0,0 +1,130 @@
+  export APP_CHAIN_ID="fxcore"
+  export APP_ADMIN_KEY_NAME="admin"
+  export APP_TEST_ACCOUNTS=3000
+  export APP_TEST_ACCOUNTS_DIR="$HOME/test_accounts"
+  if [ "$1" == "init" ]; then
+    init_genesis
+    json_processor "$APP_HOME/config/genesis.json" '.consensus_params.block.max_bytes = "22020096"'
+    json_processor "$APP_HOME/config/genesis.json" '.consensus_params.block.max_gas = "-1"'
+    json_processor "$APP_HOME/config/genesis.json" '.consensus_params.block.time_iota_ms = "100"'
+    exit 0
+  fi
+  for block_time in "block_timout_with_1s"; do
```

Shellcheck

[warning] This loop will only ever run once. Bad quoting or missing glob/expansion?

(SC2043)